src/platform_impl/linux/x11/window.rs

```diff
@@ -469,21 +472,40 @@ impl UnownedWindow {
                 leap!(window.set_icon_inner(icon.inner)).ignore_error();
             }
 
-            // Opt into handling window close
+            // Opt into handling window close and resize synchronization
```

Comment:
Is there a better source of documentation on resize synchronization?

Reply:
There's https://www.x.org/releases/X11R7.7/doc/xextproto/sync.html and https://fishsoup.net/misc/wm-spec-synchronization.html, but outside of that there's pretty much just direct implementations like Qt's.

Reply:
Do I maybe add these links to this comment?